code/datums/components/riding.dm

```diff
@@ -329,6 +329,9 @@
 		else
 			inhand.rider = riding_target_override
 		inhand.parent = AM
+		for(var/obj/item/I in user.held_items) // yes i know this sucks but these are ABSTRACT++ dumbness and i'm not adding a whole new flag for these two meme items

```

Comment:
they deserve their own flag or some form of identifier then because this will just be a thing that re-occurs if someone adds "fake" items

Reply:
Yep, if you could give them a `FAKE_ITEM`, `VISUAL_ONLY` or similar flag, that'd solve the issue.